accounts/urls.py

```diff
@@ -13,5 +13,5 @@
 
 	url(r'^register/', accounts_views.registerView, name = "register_url"),
 
-	url(r'^logout/', LogoutView.as_view(next_page="home"), name = "logout")
+	url(r'^logout/', LogoutView.as_view(next_page="home"), name = "logout"), 
 ]
```

Comment:
Agrega una línea al final de cada archivo que modifiques, creo que en este CR solo pasa en este archivo. La razón está [aquí](https://stackoverflow.com/questions/2402593/whats-the-reason-for-leaving-an-extra-blank-line-at-the-end-of-a-code-file#:~:text=It%20prevents%20you%20from%20forgetting,rid%20of%20your%20compiler%20warning.).